pkgs/top-level/aliases.nix

```
@@ -331,7 +331,7 @@ mapAliases {
   dtv-scan-tables_linuxtv = dtv-scan-tables; # Added 2023-03-03
   dtv-scan-tables_tvheadend = dtv-scan-tables; # Added 2023-03-03
   du-dust = dust; # Added 2024-01-19
-  dwfv = throw "'dwfv' has been removed due to lack of upstream maintenance";
+  dump1090 = dump1090-fa; # Added 2024-02-12
```

```suggestion
  dump1090 = dump1090-fa; # Added 2024-02-12
  dwfv = throw "'dwfv' has been removed due to lack of upstream maintenance";
```

There's no reason to delete the other alias

---

Didn't notice! Probably a typo, thanks